app/server/utils/user.py

```diff
@@ -509,15 +509,6 @@ def proccess_create_or_modify_user_request(
                 response_object = {'message': 'Transfer card not found'}
                 return response_object, 400
 
-    if additional_initial_disbursement and not additional_initial_disbursement <= current_app.config[
-        'MAXIMUM_CUSTOM_INITIAL_DISBURSEMENT']:
-        response_object = {
-            'message': 'Disbursement more than maximum allowed amount ({} {})'
-                .format(current_app.config['MAXIMUM_CUSTOM_INITIAL_DISBURSEMENT'] / 100,
-                        current_app.config['CURRENCY_NAME'])
-        }
-        return response_object, 400
-
```

Comment:
disbursements allowed greater than `MAXIMUM_CUSTOM_INITIAL_DISBURSEMENT`, but will not be automatically resolved as complete